extensions/2.0/Vendor/EXT_feature_metadata/examples/weather/weather.gltf

```diff
@@ -0,0 +1,219 @@
+          }
+        },
+        "townCollection": {
+          "featureClass": "town",
+          "featureCount": 3,
+          "properties": {
+            "name": {
+              "elementCount": 3,
+              "bufferView": 5,
+              "offsetsBufferView": 6
+            },
+            "population": {
+              "elementCount": 3,
+              "bufferView": 7
+            }
+          }
+        }
+      }
+    }
+  },
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "Scene",
+      "nodes": [
+        0
+      ]
+    }
+  ],
+  "nodes": [
+    {
+      "name": "Point Cloud",
+      "mesh": 0
+    }
+  ],
+  "meshes": [
+    {
+      "name": "Point Cloud Mesh",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "_FEATURE_ID_0": 1
+          },
+          "mode": 0,
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureMappings": [
+                {
+                  "featureCollection": "weatherCollection",
+                  "featureIds": {
+                    "constant": 0,
+                    "vertexStride": 1
+                  }
+                },
+                {
+                  "featureCollection": "townCollection",
+                  "featureIds": {
+                    "attribute": "_FEATURE_ID_0"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "accessors": [
+    {
+      "name": "Positions",
+      "bufferView": 0,
+      "byteOffset": 0,
+      "componentType": 5126,
+      "count": 1000,
+      "max": [
+        0.9984954469408285,
+        0.9919821104589452,
+        0.9970798239119962
+      ],
+      "min": [
+        -0.9968233643517341,
+        -0.9973561515575708,
+        -0.9991830031102058
+      ],
+      "type": "VEC3"
+    },
+    {
+      "name": "Town Feature ID",
+      "bufferView": 1,
+      "byteOffset": 0,
+      "componentType": 5121,
+      "count": 1000,
+      "type": "SCALAR"
+    }
+  ],
+  "bufferViews": [
+    {
+      "name": "Positions",
+      "buffer": 0,
+      "byteLength": 12000,
+      "byteOffset": 0
+    },
+    {
+      "name": "Town Feature ID",
+      "buffer": 1,
+      "byteLength": 12000,
+      "byteOffset": 0
+    },
+    {
+      "name": "Air Temperature",
+      "buffer": 1,
+      "byteLength": 4000,
+      "byteOffset": 12000
+    },
+    {
+      "name": "Air Pressure",
+      "buffer": 1,
+      "byteLength": 4000,
+      "byteOffset": 16000
+    },
+    {
+      "name": "Wind Velocity",
+      "buffer": 1,
+      "byteLength": 12000,
+      "byteOffset": 20000
+    },
+    {
+      "name": "Town Name",
+      "buffer": 1,
+      "byteLength": 26,
+      "byteOffset": 32000
+    },
+    {
+      "name": "Town Name Offsets",
+      "buffer": 1,
+      "byteLength": 16,
+      "byteOffset": 32026
+    },
+    {
+      "name": "Town Population",
+      "buffer": 1,
+      "byteLength": 6,
```

Comment:
Should this be `12`?

Reply:
No, but from our discussion it sounds like I might have declared something as a FLOAT32 when it should be a UINT16...